alchemiscale_fah/compute/models.py

```diff
@@ -0,0 +1,149 @@
+from enum import Enum, auto
+from typing import Optional
+from ipaddress import IPv4Address
+from datetime import datetime
+from enum import Enum
+
+from pydantic import BaseModel, Field, validator
+
+from alchemiscale.models import ScopedKey
+from ..utils import NonbondedSettings
+
+
+# FahAdaptiveSamplingClient models
+
+
+class CompressionTypeEnum(Enum):
+    NONE = "NONE"
+    BZIP2 = "BZIP2"
+    ZLIB = "ZLIB"
+    GZIP = "GZIP"
+    LZ4 = "LZ4"
+
+
+class JobStateEnum(Enum):
+    NEW = "NEW"
+    READY = "READY"
+    ASSIGNED = "ASSIGNED"
+    FINISHED = "FINISHED"
+    FAILED = "FAILED"
+    STOPPED = "STOPPED"
+    HELD = "HELD"
+    PROCESSING = "PROCESSING"
+
+
+class FahAdaptiveSamplingModel(BaseModel):
+    class Config:
+        use_enum_values = True
+
+
+class ProjectData(FahAdaptiveSamplingModel):
+    core_id: str = Field(..., description="The core ID.  E.g. 0xa8.")
+    contact: str = Field(
+        ..., description="Email of the person responsible for the project."
+    )
+    runs: int = Field(0, description="The number of runs.")
```

Comment:
```suggestion
    runs: NonNegativeInt = Field(0, description="The number of runs.")
```

Use the pydantic range validated number types for better validation: https://docs.pydantic.dev/2.3/usage/types/number_types/

Reply:
Thank you! Got it!